### ISSUE TITLE
IIndieStack init app `TypeError: formData is not iterable` on login

### ISSUE THREAD

**Issue by vovsemenv** [remix-run/indie-stack#105]
### What version of Remix are you using?

1.5.1

### Steps to Reproduce

init app with
```
npx create-remix@latest
```
npm install - yes
TypeScript - yes
then 
`npm run dev`
go to login page and type seeded account login/password

### Expected Behavior

Successful login

### Actual Behavior

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/20096510/173516158-d8b2f8e6-1f3c-4723-91ff-d4860c6e086c.png">

tried it multiple times - same result

envinfo:
```
  System:
    OS: macOS 12.2
    CPU: (10) arm64 Apple M1 Pro
    Memory: 464.61 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
    Yarn: 1.22.18 - ~/.nvm/versions/node/v16.15.0/bin/yarn
    npm: 8.5.5 - ~/.nvm/versions/node/v16.15.0/bin/npm
  Managers:
    Homebrew: 3.5.1 - /opt/homebrew/bin/brew
    pip3: 20.2.3 - /usr/bin/pip3
    RubyGems: 3.0.3.1 - /usr/bin/gem
  Utilities:
    Make: 3.81 - /usr/bin/make
    GCC: 13.1.6 - /usr/bin/gcc
    Git: 2.32.1 - /usr/bin/git
    Clang: 13.1.6 - /usr/bin/clang
  Servers:
    Apache: 2.4.51 - /usr/sbin/apachectl
  Virtualization:
    Docker: 20.10.14 - /usr/local/bin/docker
    Parallels: 17.1.2 - /usr/local/bin/prlctl
  IDEs:
    Nano: 2.0.6 - /usr/bin/nano
    VSCode: 1.67.2 - /usr/local/bin/code
    Vim: 8.2 - /usr/bin/vim
    Xcode: /undefined - /usr/bin/xcodebuild
  Languages:
    Bash: 3.2.57 - /bin/bash
    Java: 17.0.3 - /usr/bin/javac
    Perl: 5.30.3 - /usr/bin/perl
    Python: 2.7.18 - /usr/bin/python
    Python3: 3.8.9 - /usr/bin/python3
    Ruby: 2.6.8 - /usr/bin/ruby
  Databases:
    SQLite: 3.36.0 - /usr/bin/sqlite3
  Browsers:
    Chrome: 102.0.5005.115
    Safari: 15.3
```

create app log 
```
❯ npx create-remix@latest
? Where would you like to create your app? ./my-remix-app
? What type of app do you want to create? A pre-configured stack ready for 
production
? Which Stack do you want? (Learn more about these stacks: 
https://remix.run/stacks) Indie
? Do you want me to run `npm install`? Yes
⠇ Creating your app…(⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂) ⠴ idealTree:my-remix-app: sill idealTrenpm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated

added 1329 packages, and audited 1330 packages in 2m

235 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
? TypeScript or JavaScript? TypeScript
💿 Running remix.init script

> setup
> prisma migrate dev && prisma db seed

Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": SQLite database "data.db" at "file:./data.db?connection_limit=1"

SQLite database data.db created at file:./data.db?connection_limit=1

Applying migration `20220307190657_init`

The following migration(s) have been applied:

migrations/
  └─ 20220307190657_init/
    └─ migration.sql

Your database is now in sync with your schema.

✔ Generated Prisma Client (3.15.1 | library) to ./node_modules/@prisma/client in
 266ms


Running seed command `ts-node --require tsconfig-paths/register prisma/seed.ts` ...
Database has been seeded. 🌱

🌱  The seed command has been executed.


Environment variables loaded from .env
Running seed command `ts-node --require tsconfig-paths/register prisma/seed.ts` ...
Database has been seeded. 🌱

🌱  The seed command has been executed.
Setup is complete. You're now ready to rock and roll 🤘

Start development with `npm run dev`
💿 That's it! `cd` into "/Users/vladimirsemenov/external/my-remix-app" and check the README for development and deploy instructions!
```

**Comment by machour** [comment]
@vovsemenv the only occurence I find in remix for this error, is this comment in a PR: https://github.com/remix-run/remix/pull/361#discussion_r744027076
Can you make sure that `"DOM.Iterable"` is present in your tsconfig file?

**Comment by vovsemenv** [comment]
I surely have DOM.iterable. 
https://github.com/vovsemenv/remix-bug-reproduction there is reproduction. you can do `npm ci` and `npm run dev`
i commited `data.db` too soo you can get same enviorment

**Comment by belev** [comment]
I think this might be caused by some browser extension. I experienced the same problem with fresh setup of the indie stack with Remix version `1.6.5` using Chrome (main dev browser for me and I've got quite a few extensions) but not when I use other browsers (tried Firefox, Safari, Chrome incognito).

Noticed that this extensions are breaking it for me:
- Apollo Client Devtools
- Mokku: Mock API calls seamlessly

@vovsemenv might be worth checking your extensions and turning them off one by one and see if it's going to solve the problem like it did for me.
